engines/bastion_katello/app/assets/javascripts/bastion_katello/common/api-error-handler.service.js

```diff
@@ -0,0 +1,27 @@
+(function () {
+    'use strict';
+
+    /**
+     * @ngdoc service
+     * @name  Bastion.common.service:ApiErrorHandler
+     *
+     * @description
+     *   Provides common functionality in handling Katello/Foreman API Errors.
+     */
+    function ApiErrorHandler(translate, GlobalNotification) {
```

Comment:
Because the GET error handler below relies on an error structure that may not exist in foreman or other projects who would want to use bastion.